accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,502 @@
+| net5.0-win      | netcoreapp1..3.1                                           | WinForms + WPF and probably WinUI |
+|                 | (+everything else inherited from net5)                     |                                   |
+| Tizen, Unity... | Will follow the Xamarin model                              |                                   |
+
+***Open Issue**. In case of Xamarin, there is a discussion of whether the
+convergence with .NET Core will be a breaking change for existing NuGet packages
+(for example, there is a desire to change their namespaces). In that case, we
+should not map it to the existing TFM as that would be pointless.*
+
+***Open Issue**. Will the above scheme work for iPad?*
+
+### Mapping to properties
+
+There are three existing MSBuild properties:
+
+| Property                        | Meaning            | Examples                         |
+|---------------------------------|--------------------|----------------------------------|
+| TargetFramework (TFM)           | The short name     | `net4`, `netcoreapp3.0`          |
+| TargetFrameworkIdentifier (TFI) | The long name      | `.NETFramework` or `.NETCoreApp` |
+| TargetFrameworkVersion (TFV)    | The version number | `2`, `3.0`, `3.1`                |
+| TargetFrameworkProfile (TFP)    | The profile        | `Client` or `Profile124`         |
+
+***Open Issue**. The SDK has this logic duplicated from NuGet because they need
+to do this during evaluation where they can’t call custom targets. We could make
+this an MSBuild intrinsic, but that seems like a lot of work. Maybe we just live
+with the duplication. But bottom line is that we need to make that change in
+MSBuild too.*
+
+* [@rainersigwald](https://github.com/rainersigwald): We've made it pretty far
+  with duplicated logic, but it results in really ugly MSBuild, since it's not a
+  very expressive programming language. It also creates the potential for drift
+  between the two definitions. That said, exposing it directly would create a
+  new tight coupling between MSBuild and NuGet that hasn't historically existed.
+  It would probably require a direct dependency and update flow plus coherency
+  requirements on both .NET Core SDK and VS insertions. If the logic were in a
+  separate package (we've talked about pushing it down to the framework at
+  various times) it'd be great to just expose that. With the logic in NuGet,
+  it's reasonable either way, just different tradeoffs. I'm amenable to exposing
+  a property function, but maybe we should go down the road of not doing it at
+  first.
+
+We're going to map the new entries as follows:
+
+| Framework      | Identifier    | Version | Profile |
+|----------------|---------------|---------|---------|
+| net48          | .NETFramework | 4.8     |         |
+| net5.0         | .NETCoreApp   | 5.0     |         |
+| net5.0-android | .NETCoreApp   | 5.0     | android |
```

Comment:
This is currently problematic; profiles are a subset of a non-profiled version not a superset, and there are build errors/warnings that enforce those rules.

Reply:
We no longer user profiles.